src/content/en/showcase/2017/within.md

```diff
@@ -0,0 +1,386 @@
+interested in its potential. Today that interest manifests into a unique
+subdomain of our Web Platform, [https://vr.with.in/](https://vr.with.in/). Anyone
+with a VR-enabled browser can go to the site, click a button and throw on a
+headset to be immersed in our portfolio of VR films.
+
+*Today that includes but is not limited to Chrome on Daydream View. For
+information on your device and head-mounted display check out
+[https://webvr.info/](https://webvr.info/).*
+
+Like other Virtual Reality specific rendering environments, the web
+predominantly relies on a three-dimensional representation of a scene. This
+scene has a camera, your perspective, and any number of objects. To help manage
+this scene, camera, and objects we use a library called
+[Three.js](http://threejs.org/) which leverages the `<canvas>` element to throw
+rendering onto your computer's GPU. There are many useful Three.js add-ons to
+make your scene viewable in WebVR. The main two are
+[THREE.VREffect](https://github.com/mrdoob/three.js/blob/dev/examples/js/effects/VREffect.js)
+for creating a viewport for each eye and
+[THREE.VRControls](https://github.com/mrdoob/three.js/blob/dev/examples/js/controls/VRControls.js)
+for translating the perspective (e.g the rotation and position of the
+head-mounted display) convincingly into your scene. There are many examples of
+how to implement this. Check out the
+[Three.js WebVR examples](https://threejs.org/examples/?q=webvr)
+for ways to get started.
+
+As we got further into our exploration of WebVR we ran into an issue. If we look
+at the contents of the web, text is an integral part of it. While the majority
+of our content is video based, if you go to the
+[Within site](http://with.in/#films) text surrounds the content;
+user-interface, additional information about a film or related films are all
+constructed with text. Furthermore all of this text is created in the DOM. Our
+WebVR explorations and [https://vr.with.in/](https://vr.with.in/) are all in
+`<canvas>`.
+
+<figure>
+  <img src="/web/showcase/2017/images/within/1.jpg" class="attempt-left">
+  <img src="/web/showcase/2017/images/within/2.jpg" class="attempt-right">
+  <figcaption>
+    Text used in WebVR for <a href="https://vr.with.in/">vr.with.in</a>
+  </figcaption>
+</figure>
+
+## What are my Options?
+
+Luckily, there is work being done to make this possible. In fact in our research
+we found a number of effective ways to render text in a three-dimensional
+environment on a `<canvas />` element. Below is a matrix of a few we found
+marked with pros and cons for each:
+
+
+<table>
+  <tr>
+   <td>
+   </td>
+   <td>Resolution Independent
+   </td>
+   <td>Typographic Features
+   </td>
+   <td>Performance
+   </td>
+   <td>Ease of Implementation
+   </td>
+  </tr>
+  <tr>
+   <td><a href="https://vr.with.in/archive/text-2d-canvas/">2D Canvas Text</a>
+   </td>
+   <td>
+   </td>
+   <td>X
```

Comment:
I can't tell if X is a pro, or a con, instead, use
`<span class="compare-yes"></span>` or
`<span class="compare-no"></span>`